### PR TITLE
Djid in supports

### PR DESCRIPTION
**Pull request by zefciu**

### CODE REVIEW

**Comment by mkurek** [comment]
please add some screenshots/casts

File: src/ralph_assets/views/support.py [hunk 1]
```diff
@@ -65,7 +54,7 @@ def _save(self, request, *args, **kwargs):
             support.save(user=self.request.user)
             self.form.save_m2m()
             messages.success(self.request, self.message)
-            return HttpResponseRedirect(support.url)
+            return HttpResponseRedirect(support.get_absolute_url())
```
Comment (ar4s) [inline src/ralph_assets/views/support.py]: ``get_absolute_url``? This method doesn't exist in ``Support`` model.

File: src/ralph_assets/views/support.py [hunk 2]
```diff
@@ -98,68 +87,35 @@ def post(self, request, *args, **kwargs):
...
+            'price',
+            'created',
+        ]
+        djid_id = 'support-list'
+
+        additional_params = {
+            'autowidth': True,
+            'multiselect': True,
+        }
+
```
Comment (ar4s) [inline src/ralph_assets/views/support.py]: Too many blank lines

File: src/ralph_assets/views/support.py [hunk 3]
```diff
@@ -98,68 +87,35 @@ def post(self, request, *args, **kwargs):
...
-        ),
-    ]
-
-    def get_context_data(self, *args, **kwargs):
-        data = super(SupportList, self).get_context_data(
-            *args, **kwargs
-        )
-        data['supports'] = Support.objects.all()
-        return data
+    DjidClass= SupportListDjid
```
Comment (ar4s) [inline src/ralph_assets/views/support.py]: insert space after =

File: src/ralph_assets/views/support.py (hunk 1 above)
Comment (zefciu) [inline src/ralph_assets/views/support.py]: Corrected in models

File: src/ralph_assets/views/support.py (hunk 2 above)
Comment (zefciu) [inline src/ralph_assets/views/support.py]: Done

**Comment by ar4s** [comment]
:OK:

**Comment by xliiv** [comment]
:ok:

**Pull request merged**